core/chain_makers.go

```diff
@@ -100,7 +100,7 @@ func (b *BlockGen) SetParentBeaconRoot(root common.Hash) {
 	b.header.ParentBeaconRoot = &root
 	var (
 		blockContext = NewEVMBlockContext(b.header, b.cm, &b.header.Coinbase)
-		vmenv        = vm.NewEVM(blockContext, vm.TxContext{}, b.statedb, b.cm.config, vm.Config{})
+		vmenv        = vm.NewEVM(blockContext, b.statedb, b.cm.config, vm.Config{})
```

Comment:
In several places in the code, the `evm` is called the `vmenv`. It always trips me up a bit, takes me an extra second or two to "find the evm". I wouldn't mind changing `vmenv` to `evm` where applicable :)

Reply:
WILL DO!